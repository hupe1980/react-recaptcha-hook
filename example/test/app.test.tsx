import React from "react";
import { render } from "@testing-library/react";
import App from "../src/app";

test("renders badge", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
