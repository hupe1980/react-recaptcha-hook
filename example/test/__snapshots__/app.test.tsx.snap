// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders badge 1`] = `
<DocumentFragment>
  <span>
    This site is protected by reCAPTCHA and the Google 
    <a
      href="https://policies.google.com/privacy"
      rel="noopener noreferrer"
      target="_blank"
    >
      Privacy Policy
    </a>
     and 
    <a
      href="https://policies.google.com/terms"
      rel="noopener noreferrer"
      target="_blank"
    >
      Terms of Service
    </a>
     apply.
  </span>
</DocumentFragment>
`;
